### PR TITLE
Alibaba Cloud Support

### LINKED ISSUE #277: Enable Alicloud
1. Enable user to add shoot Alicloud cluster
2. For private cloud(currently for Alicloud), add a special flag when start Dashboard. 
- Just a suggestion. The start option name is **exclusive-provider**, the value coudl be **alicloud**
- If the value is specified for the option **exclusive-provider** and it is set alicloud, in the dashboard, only Alicloud cluster can be choosed to create.

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
Added Alibaba Cloud as additional infrastructureKind.
**Which issue(s) this PR fixes**:
Fixes #277

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Added support for Alibaba Cloud
```

### CODE REVIEW

**Comment by jia-jerry** [comment]
Hi @holgerkoser , I tested the CLs and it works now. Thank you for your help.

Would you please merge your code?

**Comment by holgerkoser** [comment]
@jia-jerry Thank you for testing. We will merge the code soon.

File: frontend/src/components/DisabledSecret.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@ limitations under the License.
 <template>
   <v-card class="mr-extra card_disabled">
     <v-toolbar :class="toolbarClass" dark>
-      <img v-if="icon === 'alibaba-cloud'" src="@/assets/alibaba-cloud.svg" width="32">
+      <img v-if="icon === 'alicloud'" src="@/assets/alicloud-white.svg" width="32">
```
Comment (petersutter) [inline frontend/src/components/DisabledSecret.vue]: rm

File: frontend/src/dialogs/CreateClusterDialog.vue [hunk 2]
```diff
@@ -720,7 +738,7 @@ export default {
       return semSort.desc(cloneDeep(this.kubernetesVersions(this.cloudProfileName)))
     },
     sortedCloudProviderKindList () {
-      return sortBy(this.cloudProviderKindList)
+      return intersection(['openstack', 'aws', 'azure', 'gcp', 'alicloud'], this.cloudProviderKindList)
```
Comment (petersutter) [inline frontend/src/dialogs/CreateClusterDialog.vue]: order like in secrets page

**Comment by jia-jerry** [comment]
@holgerkoser , everything is great for me!

**Comment by holgerkoser** [comment]
@jia-jerry The PR has been merged now is this sufficient for you or do you need a release?

**Comment by jia-jerry** [comment]
@holgerkoser yes, please.

**Comment by holgerkoser** [comment]
@jia-jerry Why do need a release? Why is the image not sufficient? We plan to create a new release on Thursday or Friday.

**Comment by jia-jerry** [comment]
@holgerkoser , Image is sufficient for me. Please schedule release for your convenience.

**Pull request merged**